api.swagger.yaml

```diff
@@ -2592,13 +2592,37 @@ paths:
         - 'blogs'
       description: 'Search for blogs. An empty array is returned if no blogs where found.'
       parameters:
+        - name: title
+          description: 'Filter on the blog title. Matches on whole words.'
```

Comment:
Just want to double check that this doesn't require any additional solr data? (see related comment below)

Reply:
Already in solr.